open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/static/README.md

```diff
@@ -0,0 +1,86 @@
+# Polymer App Toolbox - Starter Kit
```

Comment:
Do we need this file - or was it copied in as part of the install?
if to keep, it needs a license and copyright - but it isn't ours